(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'commands' is not defined"
p3
sS'layer'
p4
S'/opt/web2py/applications/tcc/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7fedc89501b0, file "/opt/web2py/applications/tcc/controllers/default.py:scan", line 1>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/opt/web2py/gluon/restricted.py", line 216, in restricted\n    exec(ccode, environment)\n  File "/opt/web2py/applications/tcc/controllers/default.py:scan", line 47, in <module>\n  File "/opt/web2py/gluon/globals.py", line 405, in <lambda>\n    self._caller = lambda f: f()\n  File "/opt/web2py/applications/tcc/controllers/default.py:scan", line 11, in scan\nNameError: global name \'commands\' is not defined\n'
p11
s.